x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/views/muti_checkbox_facets_view.test.tsx

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
There's a typo in this file - it's `muti_checkbox`, should be `multi_checkbox`

Reply:
Of course

Reply:
https://github.com/elastic/kibana/pull/88491/commits/8b0869a3f089c00bbdf8de33280c2c84d485624c